### ISSUE TITLE
"Network Error" when hosted all services in AWS FARGATE ECS

### ISSUE THREAD

**Issue by vishallanke** [reportportal/kubernetes#120]
We are using AWS FARGATE ECS to host Report Portal. We have created a Cluster. For each of the Report Portal service, we have created different Task Definition and added container/docker image for respective service. For example, we created task definition for UI, added UI docker image and started task against UI.  We created task definition for UAT, added UAT docker image and started task against UAT. Same for other services. Each Task receives  different IP Address. Therefore, each services runs using different IP Address.

Appropriate mapping is created using Environment Variables. Each service runs under a Task and has different IP Address. Postgres DB is hosted on AWS RDS. For example, UI service is hosted on http://13.127.217.114:8080/, UAT service runs under different IP Address, Gateway under different IP Address etc etc.

http://13.127.217.114:8080 opens the UI. But, after entering credentials it returns Network Error. When I analysed network tab , I found that token api gets called http://13.127.217.114:8080/uat/api/v1. However, UAT service is running inside another task having different IP Address . So, whats missing here ?

Point 2 - I also tried to create a single Cluster. Created single Task Definition. Added all the containers like UI, UAT, Index, Gateway etc. etc. Now, when I execute Task against this Task Definition, it gets a single IP Address and all the services runs under this. However, for service like Index, I found that it returns error as port 8080 is already in use. I observed that service  UI taking 8080 port. Therefore, I have decided to host each RP service under different Task as specified above.

<img width="758" alt="ALL RUNNING SERVICES" src="https://user-images.githubusercontent.com/46259109/80863808-4514cf80-8c9c-11ea-8684-96780b1b3048.PNG">

<img width="771" alt="CLUSTER" src="https://user-images.githubusercontent.com/46259109/80864083-db95c080-8c9d-11ea-95d4-0a9b04bca9cb.PNG">

**Comment by evjlobanova** [comment]
@viachka Thank you for your request. For now, we do not support AWS FARGATE ECS officially. 
So that there should be done some investigations before we can help.   

The team takes a look at this issue as soon as it has a capacity.

If this question is crucial for you, and you need our support. We can provide a paid contract. Paid requests are being implemented with the highest priority.

If you are interested in it, please drop us a letter to support@reportportal.io

**Comment by hlebkanonik** [comment]
Closing as stale (opened in 2023 or earlier). Please use the [latest Helm chart](https://github.com/reportportal/kubernetes/tree/main/reportportal) and [documentation](https://github.com/reportportal/kubernetes/blob/main/docs/README.md). If this problem persists with the current release, open a new issue with your Helm/Kubernetes versions and relevant logs. Summary of closed issues: [docs/issues-closed-until-2023.md](https://github.com/reportportal/kubernetes/blob/main/docs/issues-closed-until-2023.md).
